web-components/src/components/table/DocumentationTable.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableHead from '@material-ui/core/TableHead';
+// import TablePagination from '@material-ui/core/TablePagination';
```

Comment:
This was already commented out. Same for other commented-out code on this page

Reply:
Yes, especially now that we are making individual requests for each batch to get `tradable_supply`.